packages/rum-core/src/performance-monitoring/breakdown.js

```diff
@@ -32,9 +32,12 @@ import { PAGE_LOAD } from '../common/constants'
  * Interested events from the Navigation timing API
  */
 const pageLoadBreakdowns = [
-  ['fetchStart', 'responseEnd', 'Network'],
-  ['domLoading', 'domInteractive', 'DOM Processing'],
-  ['domInteractive', 'loadEventEnd', 'Page Render']
+  ['domainLookupStart', 'domainLookupEnd', 'DNS'],
+  ['connectStart', 'connectEnd', 'TCP'],
```

Comment:
Should we be calling this Connect rather than TCP, to account for QUIC? Or perhaps we just substitute QUIC for TCP when we identify its use?

Reply:
Might be a bit misleading for QUIC since it operates on UDP layer, But the features of QUIC includes 0-RTT which would eliminate this metric altogether. Shall we revisit this decision in the future?